### ISSUE TITLE
[0.9.0] Add the ability to add "else" in conditional block in third column

### ISSUE THREAD

**Issue by matheod** [StrangeLoopGames/EcoSuggestions#1155]
![image](https://user-images.githubusercontent.com/339611/88456899-1f096080-ce82-11ea-9f77-aeeaf5ee534f.png)

The screenshot is just to show what I am talking. Of course there is no point to put an condionnal if you don't use the second coluumn. But when you have a condition in the second column and what to do more logic in the third column, you are quickly annoyed by the impossibility to do an else and have to duplicated and invert condition which can take some time.

**Comment by johnkslg** [comment]
Conditional should no longer exist actually, you can just put the condition in the 'if' block. @olegshegai can you remove

**Comment by matheod** [comment]
But then this limit greatly what you can do (or force you to duplicated the same trigger multiple time).

For exemple, you can have in trigger chopped tree, in second column fallen, and in the third column additionnal execution based on condition like :

if tree matury > 30% :
--- if custom_stat_tree_karma >= 0:
--------custom_stat_tree_karma = custom_stat_tree_karma - 1
----if custom_stat_tree_karma < 0:
---------prevent

(and here you see why a else in third column would be usefull)

**Comment by johnkslg** [comment]
Yeah makes sense, it just gets clumsy with the UI. In this case having two conditions, 'fallen && karma' 'fallen && !karma'
The Fallen is duplicated but overall it will be much cleaner looking because nesting on the 'then' clause gets quite ugly
